### PR TITLE
Added cwm to window managers

### PR DESCRIPTION
**Pull request by paspanag**
Hi guys,

I'd like to add cwm to the available packages. I used the Linux port from chneukirchen/cwm and thus is only marked for Linux compatibility.

### CODE REVIEW

File: pkgs/applications/window-managers/cwm/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    url = "https://github.com/chneukirchen/cwm/archive/v5.6.tar.gz";
+    sha256 = "0986i83i3pg23ffzrdb3h931vwh1rp64sqy5pa9hzk0n2yxj0qq0";
+  };
+
+  buildInputs = [ libX11 libXinerama libXrandr libXft yacc pkgconfig ];
+
+  prePatch = ''sed -i "s@/usr/local@$out@" Makefile'';
+
+  meta = with stdenv.lib; {
+    description = "a lightweight and efficient window manager for X11";
```
Comment (pSub) [inline pkgs/applications/window-managers/cwm/default.nix]: `description` should start with a capital letter.

File: pkgs/applications/window-managers/cwm/default.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl, libX11, libXinerama, libXrandr, libXft, yacc, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  name = "cwm-5.6";
+
+  src = fetchurl {
```
Comment (pSub) [inline pkgs/applications/window-managers/cwm/default.nix]: Please use `fetchFromGitHub`.

File: pkgs/applications/window-managers/cwm/default.nix [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    sha256 = "0986i83i3pg23ffzrdb3h931vwh1rp64sqy5pa9hzk0n2yxj0qq0";
+  };
+
+  buildInputs = [ libX11 libXinerama libXrandr libXft yacc pkgconfig ];
+
+  prePatch = ''sed -i "s@/usr/local@$out@" Makefile'';
+
+  meta = with stdenv.lib; {
+    description = "a lightweight and efficient window manager for X11";
+    homepage = "https://github.com/chneukirchen/cwm";
```
Comment (pSub) [inline pkgs/applications/window-managers/cwm/default.nix]: Nitpick: You don't need quotes around URLs.

**Comment by paspanag** [comment]
Thanks. I've done my changes. I do another pull request in a few minutes.

**Comment by pSub** [comment]
You don't have to create a new pull request. Simply force-push to the branch corresponding the the pull request.

**Comment by pSub** [comment]
There are instructions on how to do this in the manual linked in CONTRIBUTING.md.

**Comment by paspanag** [comment]
Oh, I got it. Unfortunately, I reverted my fork, pulled from upstream, and applied my changes on top of it. I'll keep not of this in the future. Sorry about that.

**Pull request closed without merging**